config/schedule.rb

```diff
@@ -65,7 +65,7 @@
 # 3 updates to catalog-rebuild per day
 # These are 1 hour earlier than production to stagger the load on solr but ensure that
 #   no updates get lost when indices are swapped
-every 1.day, at: ["11:40pm", "9:55am", "1:55pm"], roles: [:cron_production] do
+every 1.day, at: ["1:40am", "11:55am", "3:55pm"], roles: [:cron_production] do
```

Comment:
why are you changing these? see comment.

Reply:
Thanks for the comment @hackmastera. I switched the timestamp because the indexing jobs need to happen after get_changes job. Those are scheduled at `["10:10am", "2:10pm", "10:35pm"]`.

Reply:
Ah, okay. I think it's important for rebuild to run before production, to protect against index entry loss during a swap. What if we use the times currently under `SOLR_URL` here on `SOLR_REINDEX_URL` and then make the ones on `SOLR_URL` be `["1:40am", "11:55am", "3:55pm"]`?